### PR TITLE
Adding fixed_string and addressing date bug

### PR DESCRIPTION
**Pull request by Paultagoras**
## Checklist
Delete items not relevant to your PR:
- [x] Unit and integration tests covering the common scenarios were added
- [ ] A human-readable description of the changes was provided to include in CHANGELOG
- [ ] For significant changes, documentation in https://github.com/ClickHouse/clickhouse-docs was updated with further explanations or tutorials

### CODE REVIEW

File: src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java [hunk 1]
```diff
@@ -392,6 +395,29 @@ private void doWriteColValue(Column col, ClickHousePipedOutputStream stream, Dat
...
+        if (value == null) {
+            BinaryStreamUtils.writeNull(stream);
+            return;
+        }
+
+        if (Objects.requireNonNull(columnType) == Type.FIXED_STRING) {
+            if (value instanceof byte[]) {
+                byte[] bytes = (byte[]) value;
+                if (bytes.length != length) {
+                    throw new DataException(String.format("Fixed string length mismatch: expected %d, got %d", length, bytes.length));
```
Comment (mshustov) [inline src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java:410]: What happens when the exception is thrown? Are there any tests for it?

File: src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaProxyTest.java [hunk 2]
```diff
@@ -343,6 +343,28 @@ public void schemaWithBytesTest() {
...
 
+    @Test
+    public void schemaWithFixedStringTest() {
+        Map<String, String> props = getTestProperties();
+        ClickHouseHelperClient chc = createClient(props);
+
+        String topic = "fixed-string-value-table-test";
+        ClickHouseTestHelpers.dropTable(chc, topic);
+        ClickHouseTestHelpers.createTable(chc, topic, "CREATE TABLE `%s` ( `off16` Int16, " +
+                "`fixed_string_8` FixedString(8)," +
```
Comment (mzitnik) [inline src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaProxyTest.java:354]: Why do we have 4 types of FixedString, we can make it random

File: src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java [hunk 3]
```diff
@@ -256,10 +257,9 @@ private void doWriteDates(Type type, ClickHousePipedOutputStream stream, Data va
                 if (value.getFieldType().equals(Schema.Type.INT32) || value.getFieldType().equals(Schema.Type.INT64)) {
                     if (value.getObject().getClass().getName().endsWith(".Date")) {
                         Date date = (Date) value.getObject();
-                        long epochSecond = date.toInstant().getEpochSecond();
-                        BinaryStreamUtils.writeUnsignedInt32(stream, epochSecond);
+                        BinaryStreamUtils.writeUnsignedInt32(stream, date.toInstant().getEpochSecond());
```
Comment (mzitnik) [inline src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java:260]: Do we cover this in the tests?

File: src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java [hunk 4]
```diff
@@ -256,10 +257,9 @@ private void doWriteDates(Type type, ClickHousePipedOutputStream stream, Data va
...
                     } else {
-                        BinaryStreamUtils.writeUnsignedInt32(stream, (Long) value.getObject());
+                        BinaryStreamUtils.writeUnsignedInt32(stream, Long.parseLong(String.valueOf(value.getObject())));
```
Comment (mzitnik) [inline src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java:262]: Same here?

**Comment by mzitnik** [comment]
@Paultagoras what was the date bug?

**Review by mzitnik — CHANGES_REQUESTED** [review]

**Review by mzitnik — APPROVED** [review]

**Pull request merged**